### PR TITLE
win_virtio_driver_installer: add threashold for balloon function

### PR DESCRIPTION
**Pull request by xiagao**
ID: 2211373
For balloon check during installer test, threshold should be set.

### CODE REVIEW

**Comment by xiagao** [comment]
@leidwang  Could you help to review, thanks.

File: qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg [hunk 1]
```diff
@@ -121,15 +123,15 @@
...
-                   variants:
+                    images += " stg0"
+                    image_name_stg0 = "images/stg0"
+                    image_size_stg0 = 5G
+                    remove_image_stg0 = yes
+                    force_create_image_stg0 = yes
+                    driver_test_names = 'iozone'
+                    driver_test_params = {'images': 'stg0'}
+                    iozone_cmd_opitons = " -azR -r 64k -n 512M -g 1G -M -I -i 0 -i 1 -b iozone.xls -f %s:\testfile"
+                    variants:
```
Comment (leidwang) [inline qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg:134]: Seems these code change was by mistake?

File: qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg [hunk 2]
```diff
@@ -73,6 +73,8 @@
             balloon = balloon0
             balloon_dev_devid = balloon0
             balloon_dev_add_bus = yes
+            guest_compare_threshold = 300
```
Comment (leidwang) [inline qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg:76]: Could you give some introduction about why we need a threshold?Why we do not need it before?

File: qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg (hunk 1 above)
Comment (xiagao) [reply qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg:134]: NO. The original config lack one space, therefore CI check failed.

File: qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg (hunk 2 above)
Comment (xiagao) [reply qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg:76]: It's balloon specific config, you can check balloon_check related cases. If there's no threshold set, this case may fail sometimes(windows os' memory is not as controlled as Linux os).

File: qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg (hunk 1 above)
Comment (leidwang) [reply qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg:134]: Ok.

File: qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg (hunk 2 above)
Comment (leidwang) [reply qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg:76]: Okay,please provide test results here.

**Comment by xiagao** [comment]
(1/1) Host_RHEL.m9.u3.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.win_virtio_driver_install_by_installer.driver_install.single_driver.with_balloon.q35: STARTED
 (1/1) Host_RHEL.m9.u3.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.win_virtio_driver_install_by_installer.driver_install.single_driver.with_balloon.q35: PASS (633.29 s)
@leidwang  Here are the latest results,thx

File: qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg (hunk 2 above)
Comment (leidwang) [reply qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg:76]: > (1/1) Host_RHEL.m9.u3.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.win_virtio_driver_install_by_installer.driver_install.single_driver.with_balloon.q35: STARTED (1/1) Host_RHEL.m9.u3.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.win_virtio_driver_install_by_installer.driver_install.single_driver.with_balloon.q35: PASS (633.29 s) @leidwang Here are the latest results,thx

Okay, LGTM.

Just have a small question about the value of `guest_compare_threshold`, I see in some cases, the value is 150,and in some other cases, the value is 300. Could you explain why? What is its impact? Thanks！

Comment (xiagao) [reply qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg:76]: As you can searched, only balloon_minimum test case is set to 150, most test cases are set to 300, I thought  the previous one also should be set to 300, and the proper guest_compare_threshold value is talked for a long time, if you're interested you can search in internal bzs, there are several bugs talking about it.

**Review by leidwang — APPROVED** [review]

**Comment by leidwang** [comment]
@xiagao Please @ someone to review this patch?

**Comment by xiagao** [comment]
@vivianQizhu Could you help to review this patch? Thanks.

**Review by vivianQizhu — APPROVED** [review]

**Pull request merged**